roles/network_plugin/calico/defaults/main.yml

```diff
@@ -80,6 +80,11 @@ calico_iptables_lock_timeout_secs: 10
 # Choose Calico iptables backend: "Legacy", "Auto" or "NFT" (FELIX_IPTABLESBACKEND)
 calico_iptables_backend: "Legacy"
 
+# Calico Wireguard support
+calico_wireguard_enabled: false
+calico_wireguard_packages: []
+calico_wireguard_repo: https://download.copr.fedorainfracloud.org/results/jdoss/wireguard/epel-{{ ansible_distribution_major_version }}-$basearch/
```

Comment:
Nit-pick: Should this be `calico_wireguard_fedora_repo` or `calico_wireguard_epel_repo`?

Reply:
This is used by CentOS and some derivatives (think Alma and Amazon) and is not tied to EPEL. This repo contains the `wireguard-dkms` kernel module while the `wireguard-tools` comes from EPEL (seems that non-Debian distro's are all over the map on this one).

Debian, Ubuntu, OpenSUSE Leap and Oracle Linux actually contain both `wireguard-tools` and `wireguard-dkms` packages in their main repos and are a bit more sound.

This is my reasoning for the variable naming.